src/main/java/hudson/remoting/Asynchronous.java

```diff
@@ -31,7 +31,7 @@
  * // this method actually gets executed. For example, if you invoke two async
  * // calls, they may execute in the reverse order.
  * foo.zot();
- *
+ * </pre>
```

Comment:
Thanks for catching. Makes sense to run Javadoc  linter in as a part of the build